packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -76,7 +76,11 @@ RadioGroup.propTypes /* remove-proptypes */ = {
   /**
    * The default value. Use when the component is not controlled.
    */
-  defaultValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  defaultValue: PropTypes.oneOfType([
```

Comment:
This type is wrong. An array of string and number?

Also, string and number are not the only types accepted. Anything that has a valid `.toString` implementation works.

Reply:
Can you check the code again, the changes does not include this file.

Reply:
We used to have a fix for

<img width="350" alt="Capture d’écran 2021-06-21 à 13 50 54" src="https://user-images.githubusercontent.com/3165635/122757270-b8f6aa80-d297-11eb-8a57-420c6c31e160.png">

https://deploy-preview-26772--material-ui.netlify.app/api/radio-group/#props

Now, the wrong prop-types is back in.

Reply:
Are you suggest that the `defaultValue` type should be any?

Reply:
I think that `array<string>` has no place here. I wouldn't be against any for simplicity, or `number | string` if we want to be very strict on the types.

Reply:
Ok, we can solve these two issues in isolation. Moved to #26945.